src/main/java/edu/harvard/iq/dataverse/export/openaire/OpenAireExportUtil.java

```diff
@@ -1134,12 +1134,10 @@ public static void writeAccessRightsElement(XMLStreamWriter xmlw, DatasetVersion
         // check if getLicense() method contains CC0
         // check if getTermsOfUse() method starts with http://
         writeRightsHeader(xmlw, language);
-        if (StringUtils.isNotBlank(datasetVersionDTO.getLicense())) {
-            if (StringUtils.containsIgnoreCase(datasetVersionDTO.getLicense(), "cc0")) {
-                xmlw.writeAttribute("rightsURI", TermsOfUseAndAccess.CC0_URI);
-                if (StringUtils.isNotBlank(datasetVersionDTO.getTermsOfUse())) {
-                    xmlw.writeCharacters(datasetVersionDTO.getTermsOfUse());
-                }
+        if (datasetVersionDTO.getLicense() != null) {
+            if (!StringUtils.containsIgnoreCase(datasetVersionDTO.getLicense().getLabel(), "custom")) {
```

Comment:
Why wouldn't a custom license get included?

Reply:
@qqmyers I think it does get included? If it's not a custom license it writes the normal license otherwise it uses the terms of use specified in the custom license.

Reply:
I see that the Terms of Use get included, but I guess I was expecting the custom license URI to be included. I don't think this is a big deal but I don't think we want to continue promoting a terms of use starting with http to look the same as an internally selected license - that would be a way to include unapproved licenses. Others may know more about what's expected/best practice for the OpenAire export but without furtehr input, I'd suggest including the custom license URI for the rightsURI attribute and always use writeCharacters() for the terms of use, regardless of it's format (assuming the rightURI attribute and a text value are allowed at the same time - do we know of any restriction?)